pyproject.toml

```diff
@@ -34,7 +34,7 @@ keywords = [
 ]
 requires-python = ">=3.9"
 dynamic = ["version"]
-license-files = { paths = ["LICENSE"] }
+license = {file = "LICENSE"}
```

Comment:
drive-by fix which the latest ruff release flags

Reply:
Related
- https://github.com/vega/altair/discussions/3687#discussioncomment-11351453

Reply:
thanks!

i took this from https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license